nservicebus/scalability-and-ha/distributor/index.md

```diff
@@ -73,6 +73,7 @@ For more information about Pub/Sub in a distributor scenario see [What the distr
 
 ## Distributor configuration
 
+NOTE: As of Version 6 NServiceBus no longer supports running an embedded distributor. Because of that configuration sections like `MasteNodeConfig` are obsoleted. Read more about [Upgrading a Distributor-based scaled out endpoint to V6](/samples/scaleout/distributor-upgrade/)
```

Comment:
Because distributor is needed for compat with 5 or lower, we still need this article. People will still need to configure their distributor in front of a brand new V6 cluster if they have upstream V5 or V4 endpoints.